File: BOJ/9498. 시험 성적/solution.cc
// https://www.acmicpc.net/problem/9498
#include <iostream>

using namespace std;

int main() {
    int s;
    cin >> s;
    cout << (s >= 90 ? 'A' : s >= 80 ? 'B' : s >= 70 ? 'C' : s >= 60 ? 'D' : 'F');
    return 0;
}

File: BOJ/9498. 시험 성적/solution.js
// https://www.acmicpc.net/problem/9498
const s = +require('fs').readFileSync('/dev/stdin').toString();
console.log(s >= 90 ? 'A' : s >= 80 ? 'B' : s >= 70 ? 'C' : s >= 60 ? 'D' : 'F');